spec/support/capybara.rb

```diff
@@ -1,9 +1,9 @@
 require 'capybara/rspec'
-require 'webdrivers/chromedriver'
+
 require 'rack_session_access/capybara'
 
 Capybara.register_driver :headless_chrome do |app|
-  browser_options = Selenium::WebDriver::Chrome::Options.new
+  browser_options = driver = Selenium::WebDriver::Chrome::Options.new
```

Comment:
looks like maybe we don't need the extra `device` assignment?
```suggestion
  browser_options = Selenium::WebDriver::Chrome::Options.new
```

Reply:
whoops yeah this was a typo, thanks